.github/workflows/helm-version-update.yaml

```diff
@@ -0,0 +1,57 @@
+name: Build
+on:
+  schedule:
+    - cron: '0 8 * * 1'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  HELM_VERSION: v3.11.1
+
+jobs:
+  permissions:
+    pull-request: write
+
+  ensure-versions:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.5.0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3.5
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      - name: Update Versions
+        run: |
+          source .github/tests/versions.sh
+          source .github/tests/repos.sh
+          set | grep HELM_REPO | awk -F= '{print $1}' | while read REPO; do
```

Comment:
Instead of grepping for the `HELM_REPO` prefix is there a way we can read all the repos into an array and then step through the array?

Reply:
I think we can do this with a bash script.

Reply:
reworked.